frontend/src/tests/pages/PlayPage.test.js

```diff
@@ -89,8 +89,6 @@ describe("PlayPage tests", () => {
 
         await waitFor(() => expect(axiosMock.history.put.length).toBe(1));
 
-        expect(mockToast).toBeCalledWith("Cow bought!");
```

Comment:
Looks like this PR has code from another PR in it... one on which it does not depend.

If these are two independent PRs, we should probably keep the code for them separate.

Reply:
This code is from PR #16 and probably shouldn't be here.

Reply:
Fixed with rebase.